ui/html/ui-tools.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Lamassu Machine Dev Tools</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <style>
+    body {
+      background-color: #82dda3;
+      font-family: 'MuseoSans', sans-serif;
+    }
+    main {
+      max-width: 42rem;
+      margin: 3rem auto;
+    }
+    button, input {
+      background-color: #fff;
+      border: none;
+      padding: 0.5rem 1rem;
+      border-radius: 0.5rem;
+      font-size: 1.2rem;
+    }
+    button { cursor: pointer; }
+    button:hover { background-color: #f0f0f0; }
+    button:focus, input:focus { outline: 2px solid #3d65ec; }
+    .container {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 1rem;
+      background-color: #66ba85;
+      padding: 2rem;
+      border-radius: 0.5rem;
+    }
+    #custom-bill-input { width: 8rem; }
+    input { flex-grow: 1; }
+    .in { margin-left: auto; }
```

Comment:
Standard font choice for addresses 

```suggestion
    .in { margin-left: auto; }
    #pairing-token input, #wallet-addresses input {
      font-family: monospace;
      font-size: 1rem
    } 
```